helm/cloud-chart/values.yaml

```diff
@@ -9,6 +9,16 @@ namespace: fogflow
 # are running all the time for the deployment
 replicaCount: 1
 
+readinessProbe:
```

Comment:
Broker and master pods are not coming up after this changes. Need to verify once before merging.